_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -3500,7 +3513,12 @@ mfxStatus ImplementationAvc::AsyncRoutine(mfxBitstream * bs)
 
                 mfxStatus sts = MFX_ERR_NONE;
 
-                //printf("Execute: %d, type %d, qp %d\n", task->m_frameOrder, task->m_type[0], task->m_cqpValue[0]);
+                //printf("Execute: %d, type %d, qp %d\n", task->m_frameOrder, task->m_type[0], task->m_cqpValue[0]);#if defined(MXF_ENABLE_MCTF_IN_AVC)
+#if defined(MXF_ENABLE_MCTF_IN_AVC)
+                if(task->m_handleMCTF.first && task->m_bFrameReady)//Intercept encoder so MCTF denoised frame can be fed at the right moment.
```

Comment:
What `task->m_bFrameReady` is doing here? why needed?

Reply:
This is redundant I will remove m_bFrameReady from this statement.